ui/tests/integration/list-pagination-test.js

```diff
@@ -20,20 +20,20 @@ module('Integration | Component | list pagination', function(hooks) {
   test('the source property', async function(assert) {
     this.set('source', list100);
     await render(hbs`
-      {{#list-pagination source=source as |p|}}
+      <ListPagination @source={{source}} as |p|>
         <span class="page-info">{{p.currentPage}} of {{p.totalPages}}</span>
-        {{#p.first}}<span class="first">first</span>{{/p.first}}
-        {{#p.prev}}<span class="prev">prev</span>{{/p.prev}}
+        <p.first><span class="first">first</span></p.first>
+        <p.prev><span class="prev">prev</span></p.prev>
```

Comment:
I think at some point we should refactor contextual components to yield components using the same capitalized camelcase as is now standard for component invocation.

Reply:
Agreed!